src/write.c

```diff
@@ -231,6 +231,9 @@ typedef struct avifEncoderData
     uint16_t primaryItemID;
     avifBool singleImage; // if true, the AVIF_ADD_IMAGE_FLAG_SINGLE flag was set on the first call to avifEncoderAddImage()
     avifBool alphaPresent;
+    // Cached encoded image from avifEncoderAddImage() when avifEncoder.targetSize>0.
```

Comment:
Nit: Not sure if you prefer "x>0", with no spaces around the `>` sign, in the comments. There is another instance in line 1294.